Fitness/Controllers/ClassListViewController.swift

```diff
@@ -70,6 +70,7 @@ class ClassListViewController: UIViewController {
 
     init() {
         super.init(nibName: nil, bundle: nil)
```

Comment:
Ik u didn't write this but can we change this to the normal init? also y do we do this over handling this in viewDidLoad

Reply:
wait what do you mean by normal init?

Reply:
we need to initialize the collection views here because theres logic in `HomeViewController` that configures stuff in `ClassListViewController` which could happen before the user goes to `ClassListViewController` in the app

Reply:
`super.init()` rather than calling the one w `nibName` and `bundle` and passing in `nil` arguments

Reply:
the default one for view controllers is `nibName, bundle`. you can't call `super.init()`